tests/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rockylinux:8
+
+ENV FTP_USER=admin \
+    FTP_PASS=random \
+    LOG_STDOUT=false \
+    ANONYMOUS_ACCESS=false \
+    UPLOADED_FILES_WORLD_READABLE=false \
+    CUSTOM_PASSIVE_ADDRESS=false
+
+RUN \
+  yum clean all && \
+  yum install -y vsftpd ncurses && \
+  yum clean all
+
+COPY container-files /
+
+EXPOSE 20-21 21100-21110
+
+ENTRYPOINT ["/bootstrap.sh"]
```

Comment:
This `Dockerfile` is for the fake FTP server and used on the GitHub action.

Reply:
That okey, I Think all the files that related to the integration tests must be within the `integration` folder, what you say ?

Reply:
Agree. `Dockerfile` and `container-files` are moved to `integration` directory.

Reply:
If the `tests/config.php` is not used anymore as long as we use a fake FTP server, I think both `tests/bootstrap.php` and `tests/config.php` should be removed ?

Reply:
The `tests/config.php` is defined with the FTP server connection information. And the `Dockerfile` and `container-files` are for FTP server confguration.

Reply:
Sorry, but I don't really understand! What is the file that uses the `config.php` file settings for the fake server configuration ?

Reply:
The `config.php` is not used for fake server configuration. And it lets integration tests know the FTP server connection information. The `config.php` records the FTP server connection, such as port number, IP address and so on.

I keep this file because it will be changed at some days and it will be convenient to change these settings easily.

Reply:
I understand now thank you!